webui/react/src/services/api.ts

```diff
@@ -18,6 +18,7 @@ import { serverAddress } from 'utils/routes';
 import { isExperimentTask } from 'utils/task';
 
 export const detAuthApi = new DetSwagger.AuthenticationApi(undefined, serverAddress());
+export const detExperimentsApi = DetSwagger.ExperimentsApiFetchParamCreator();
```

Comment:
nit: these two are bit different one contains the actual API calls the other just generated the fetch parameters. 
IMO we might want to reserve `detExperimentsApi` for 
`new DetSwagger.ExperimentsApi(undefined, serverAddress());`